packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -152,12 +146,12 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
     if (item.iconOnly && itemText !== undefined) {
       return (
         <TooltipHost content={itemText} {...item.tooltipHostProps}>
-          <CommandButtonType {...commandButtonProps as IButtonProps} />
+          <CommandButtonType {...commandButtonProps as IButtonProps} defaultRender={CommandButtonType} />
```

Comment:
Shouldn't this be `CommandBarButton`? Otherwise the caller might accidentally recursve themselves.

Reply:
Just updated